src/services/ethereum/factory.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import EthereumService from './index';
+import CustomSigner from './CustomSigner';
+
+/**
+ * @desc Factory for the Ethereum service based on EthJS.
+ * @param {Array<any>} config Array with: private_key Private Key base for the services, provider_type Type of Network
+ * @returns {{wallet: ethers.Wallet, service: EthereumService}} The service
+ */
+export default function factory(config: {private_key: string, provider_type: string}) {
```

Comment:
Please use lowerCamelCase for naming, so `privateKey` and `providerType`.

Reply:
changed, but @albertoestarrona you might need to change it when you pass in the variables too